### PR TITLE
Added debug mode

### PR DESCRIPTION
**Pull request by Hbrinj**
Added debug as a maven property to enable the raw HTTP requests sent to stubby to be viewed

### CODE REVIEW

File: src/main/java/com/testingsyndicate/maven/plugin/stubby/ServerManager.java [hunk 1]
```diff
@@ -166,6 +176,11 @@ Builder mute(Boolean mute) {
             return this;
         }
 
+        Builder debug(Boolean debug) {
+            this.debu = debug;
```
Comment (goughy000) [inline src/main/java/com/testingsyndicate/maven/plugin/stubby/ServerManager.java]: typo?

**Review by goughy000 — CHANGES_REQUESTED** [review]
Can you use space indentation throughout please?

**Comment by Hbrinj** [comment]
Should be all fixed, thanks, good catch!

**Review by goughy000 — DISMISSED** [review]
LGTM

**Comment by goughy000** [comment]
```
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /Users/goughy/temp/stubby-maven-plugin/src/main/java/com/testingsyndicate/maven/plugin/stubby/ServerManager.java:[39,29] cannot find symbol
  symbol:   variable debug
  location: variable builder of type com.testingsyndicate.maven.plugin.stubby.ServerManager.Builder
[ERROR] /Users/goughy/temp/stubby-maven-plugin/src/main/java/com/testingsyndicate/maven/plugin/stubby/ServerManager.java:[180,17] cannot find symbol
  symbol: variable debug
[INFO] 2 errors 
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
```

**Review by goughy000 — APPROVED** [review]

**Pull request merged**